libcodechecker/source_code_comment_handler.py

```diff
@@ -185,9 +185,8 @@ def get_source_line_comments(self, bug_line):
         for a bug line.
         """
         source_file = self.__source_file
-        LOG.debug("Checking for source code comments in the source file '{0}'"
-                  "at line {1}".format(self.__source_file,
-                                       bug_line))
+        LOG.debug("Checking for source code comments in the source file '%s'"
+                  "at line %s", self.__source_file, bug_line)
```

Comment:
As far as I've seen, this `bug_line` really is a string, but I wouldn't have counted on that.